src/main/scala/snabbdom/h.scala

```diff
@@ -50,7 +50,7 @@ object Hooks {
 
 @ScalaJSDefined
 trait DataObject extends js.Object {
-  val attrs: js.Dictionary[String | Boolean]
+  val attrs: js.Dictionary[Attribute.Value]
```

Comment:
`Attribute` should use the type `Value` from the snabbdom-facade, not the other way round. This makes it possible to move the facade into an sbt subproject later.

Reply:
I agree, but the snabbdom facade already [depends on Attribute anyhow](https://github.com/OutWatch/outwatch/blob/master/src/main/scala/snabbdom/h.scala#L115). We should fix this but maybe not in this PR. I have opened a new issue https://github.com/OutWatch/outwatch/issues/94.